# This service is used to upload or update an image thumbnail to an existing
# object on the interoperability server.

# Object ID.
uint64 id

# Image thumbnail.
sensor_msgs/Image image

---

# Whether the request was successful.
bool success
